aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/plugins/categories_list.html

```diff
@@ -1,5 +1,10 @@
 {% load i18n apphooks_config_tags %}
  <div class="aldryn aldryn-jobs aldryn-jobs-categories">
+{% if plugin_configuration_error %}
```

Comment:
@cyberstar be aware that this changed in: https://github.com/aldryn/aldryn-jobs/pull/61